portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,77 +31,65 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.gson.Gson;
-import com.google.appengine.api.datastore.Key;
-import com.google.appengine.api.datastore.KeyFactory;
 
 /** Servlet that encapsulates some data from training exercises. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+
   /* Do Get function to fetch and list the todo list items onto the home page*/
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException 
+  {
```

Comment:
don't add a line break before an opening brace (same elsewhere in the code)
https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style

Reply:
fixed